### PR TITLE
Add a page about alerts CRUD

### LINKED ISSUE #60: Add a page for the alerts
Related to Scalingo/api#1023

### PR DESCRIPTION
**Pull request by EtienneM**
Lack:
- [x] remind_every, send_when_below in add and update
- [x] container_type in get

Fix #60

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ services:
   web:
     build: .
     ports:
-      - 4000:4000
+      - 4002:4000
```
Comment (johnsudaar) [inline docker-compose.yml]: Why ? Is it conflicting with another project ?

File: markdown/alerts.md [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+		"id": "al-297f1134-8576-45ac-ab58-2100b6807683",
+		"app_id": "5ab1128fdccb8b0001111963",
+		"created_at": "2018-03-20T14:56:06.27014439Z",
+		"updated_at": "2018-03-20T14:56:06.27014439Z",
+		"container_type": "web",
+		"disabled": false,
+		"metric": "memory",
+		"limit": 0.8,
+		"send_when_below": false,
+		"metadata": {
```
Comment (johnsudaar) [inline markdown/alerts.md]: Should we really send the medatadata to the users. If i recall correctly this is only a private thing (used for the communication between the alerter and the autoscaler).

Comment (EtienneM) [reply markdown/alerts.md]: That's a god question. I don't see the harm to send it back to the user

Comment (Soulou) [reply markdown/alerts.md]: It's not about harming, it's more about, is it useful in our public API?

Comment (EtienneM) [reply markdown/alerts.md]: If someone lists all his application's alerts, he can see which are for the autoscaler, and which he did defined

Comment (Soulou) [reply markdown/alerts.md]: That's a good argument. What if the user DELETE it ?

Comment (EtienneM) [reply markdown/alerts.md]: He breaks the autoscaler

Comment (Soulou) [reply markdown/alerts.md]: Shouldn't we avoid this?

Comment (EtienneM) [reply markdown/alerts.md]: Definitely! I can add a check on the API in the alerts#destroy action to return a bad request if a user tries to delete an autoscaler alert?

Comment (EtienneM) [reply markdown/alerts.md]: Continue to discuss [here](https://github.com/Scalingo/api/issues/1140)

**Pull request merged**